File: RAI/EjemploBanco/JavaImplementation/Cuenta.java
public class Cuenta extends Producto{
    private int saldo;

    public Cuenta(int id, int saldo){
        super(id);
        this.saldo = saldo;
    }

    public int getSaldo(){
        return this.saldo;
    }

    public String toString(){
        return super.toString() + "Cuenta: " + this.saldo;
    }
}

File: RAI/EjemploBanco/python_implementation/Cuenta.py
from Producto import Producto

class Cuenta(Producto):
    def __init__(self, id, saldo):
        super().__init__(id)
        self.saldo = saldo

    def get_saldo(self):
        return self.saldo

    def __str__(self):
        return super().__str__() + "Cuenta: " + str(self.saldo)